functorch/functorch/csrc/Constants.h

```diff
@@ -8,26 +8,16 @@
 #include <c10/core/DispatchKey.h>
 
 // This file contains aliases for dispatch keys related to functorch.
-// We created these initially just in case we needed to change the names
-// of the keys in core but that is no longer relevant, so this file can
-// be deleted after refactoring.
+// The names were long so we aliased them.
 
 namespace at {
 namespace functorch {
 
-#define FT_BATCHED_KEY FuncTorchBatched
-#define FT_VMAP_MODE_KEY FuncTorchVmapMode
-#define FT_GRAD_WRAPPER_KEY FuncTorchGradWrapper
-#define FT_DYNAMIC_LAYER_FRONT_MODE_KEY FuncTorchDynamicLayerFrontMode
-#define FT_DYNAMIC_LAYER_BACK_MODE_KEY FuncTorchDynamicLayerBackMode
-#define FT_PYTHON_KEY FuncTorchPython
-
-constexpr auto kBatchedKey = c10::DispatchKey::FT_BATCHED_KEY;
-constexpr auto kVmapModeKey = c10::DispatchKey::FT_VMAP_MODE_KEY;
-constexpr auto kGradWrapperKey = c10::DispatchKey::FT_GRAD_WRAPPER_KEY;
-constexpr auto kDynamicLayerFrontModeKey = c10::DispatchKey::FT_DYNAMIC_LAYER_FRONT_MODE_KEY;
-constexpr auto kDynamicLayerBackModeKey = c10::DispatchKey::FT_DYNAMIC_LAYER_BACK_MODE_KEY;
-//# constexpr auto kPythonKey = c10::DispatchKey::FT_PYTHON_KEY;
+constexpr auto kBatchedKey = c10::DispatchKey::FuncTorchBatched;
```

Comment:
Why do we have these since it looks like now we're using the actual dispatch name (i.e. "FuncTorchBatched" instead of "kBatchedKey")?

Reply:
`kBatchedKey` is shorter than typing out `DispatchKey::FuncTorchBatched`, though I guess not by far. Do you think we should just go with `DispatchKey::FuncTorchBatched` ?